apispec/core.py

```diff
@@ -106,11 +113,22 @@ def to_dict(self):
             'swagger': SWAGGER_VERSION,
             'info': self.info,
             'definitions': self._definitions,
+            'parameters': self._parameters,
             'paths': self._paths,
         }
         ret.update(self.options)
         return ret
 
+    def add_parameter(self, _name, _in, **kwargs):
```

Comment:
Not sure about the argument names here. Maybe `name` and `display_name`? is there a need to pull off `kwargs` rather than including the arguments explicitly in the function declaration? 

Also, so a trailing underscore to avoid clashing with a keyword, i.e. `in_`, as per PEP8.